src/java/htsjdk/samtools/cram/structure/ContainerHeaderIO.java

```diff
@@ -80,7 +88,7 @@ public int writeContainerHeader(final int major, final Container container, fina
 
         if (major >= 3) {
             outputStream.write(crc32OutputStream.getCrc32_LittleEndian());
-            length += 4 * 8;
+            length += 4 ;
```

Comment:
Looking at the htslib code, I'm assuming this code was originally ported incorrectly (see https://github.com/samtools/htslib/blob/develop/cram/cram_io.c#L3159), but it never mattered  until we made the change in this PR to switch to writing v3 files because this code path was never executed.